src/ansible_compat/config.py

```diff
@@ -77,5 +84,50 @@ def ansible_version(version: str = "") -> Version:
     return Version(version)
 
 
+class AnsibleConfig(_UserDict):  # pylint: disable=too-many-ancestors
+    """Interface to query Ansible configuration.
+
+    This should allow user to access everything provided by `ansible-config dump` without having to parse the data himself.
+    """
+
+    _aliases = {
+        'COLLECTIONS_PATHS': 'COLLECTIONS_PATH',  # 2.9 -> 2.10+
+        'COLLECTIONS_PATH': 'COLLECTIONS_PATHS',  # 2.10+ -> 2.9
+    }
+
+    def __init__(self) -> None:
+        """Load config dictionary."""
+        super().__init__()
+        env = os.environ.copy()
+        # Avoid possible ANSI garbage
+        env["ANSIBLE_FORCE_COLOR"] = "0"
+
+        config = subprocess.check_output(
+            ["ansible-config", "dump"], universal_newlines=True, env=env
+        )
+        for match in re.finditer(
+            r"^(?P<key>[A-Za-z0-9_]+).* = (?P<value>.*)$", config, re.MULTILINE
+        ):
+            key = match.groupdict()['key']
+            value = match.groupdict()['value']
+            try:
+                self[key] = eval(value)  # pylint: disable=eval-used
```

Comment:
I would really avoid using `eval` in anything than testing code.

Reply:
Believe me that I did not want to use eval either but I did not know exactly what to use. I am open to suggestions here.

Reply:
I added #11 for tracking this bit in follow-up.